src/integration/java/io/pinecone/AfterAllTestsListener.java

```diff
@@ -0,0 +1,22 @@
+package io.pinecone;
+
+import io.pinecone.helpers.IndexManagerSingleton;
+import org.junit.platform.launcher.TestExecutionListener;
+import org.junit.platform.launcher.TestPlan;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AfterAllTestsListener implements TestExecutionListener {
+    private static final Logger logger = LoggerFactory.getLogger(AfterAllTestsListener.class);
+
```

Comment:
Should we call this class the `CleanupAllTestResourcesListener`?

Reply:
Yes, I think we should. Updated.